File: 867_Transpose Matrix.java
class Solution {
   public int[][] transpose(int[][] matrix) {
      int res[][] = new int[matrix[0].length][matrix.length];
      for (int i = 0; i < matrix.length; i++) {
         for (int j = 0; j < matrix[0].length; j++) {
            res[j][i]=matrix[i][j];
            // matrix[j][i]=t; 
         }
      }
      return res;
   }
}

File: 867_Transpose Matrix.py
from typing import List


class Solution:
    def transpose(self, matrix: List[List[int]]) -> List[List[int]]:
        res = []
        for i in range(len(matrix[0])):
            row = []
            for j in range(len(matrix)):
                row.append(matrix[j][i])
            res.append(row)

        return res
